### PR TITLE
Fix spacing between layers and topics

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/TopicMenu/TopicMenu.scss [hunk 1]
```diff
@@ -95,6 +95,7 @@
     margin: 2px 0 0 20px;
 
     .rs-layer-tree {
+      margin-top: -1px;
```
Comment (oterral) [inline src/components/TopicMenu/TopicMenu.scss]: do we really need this one ?

Comment (edellucien) [reply src/components/TopicMenu/TopicMenu.scss]: To ensure the pixel precise distance between layer items and topics

Comment (oterral) [reply src/components/TopicMenu/TopicMenu.scss]: Why not putting margin-top: 1px to the wkp-layer-tree class instead dof 2px ?

Comment (oterral) [reply src/components/TopicMenu/TopicMenu.scss]: maybe remove the wkp-layer-tree element , it seems useless.

Comment (edellucien) [reply src/components/TopicMenu/TopicMenu.scss]: It works as well indeed

**Pull request merged**